frontend/trpc/routes/equityGrants.ts

```
@@ -171,14 +171,15 @@ export const equityGrantsRouter = createRouter({
           vestingFrequencyMonths: z.string().nullable(),
-          docusealTemplateId: z.string(),
+          docusealTemplateId: z.string().optional(),
+          internalTemplateId: z.bigint().optional(),
         }),
     )
     .mutation(async ({ input, ctx }) => {
       if (!ctx.companyAdministrator) throw new TRPCError({ code: "FORBIDDEN" });
       const template = await db.query.documentTemplates.findFirst({
         where: and(
-          eq(documentTemplates.externalId, input.docusealTemplateId),
+          eq(documentTemplates.externalId, input.docusealTemplateId!),
```

ESLint

[error] Forbidden non-null assertion.

(@typescript-eslint/no-non-null-assertion)